File: Práctica 1/S1/S1R/lib/FactoriaMontana.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
require_relative "BicicletaMontana"
require_relative "CarreraMontana"

module Ruby
  class FactoriaMontana < FactoriaCarrerayBicicleta
    def crearCarrera(id)
      carrera = CarreraMontana.new(id)
      return carrera
    end
    
    def crearBicicleta(id)
      bicicleta = BicicletaMontana.new(id)
      return bicicleta
    end
  end
end

File: Práctica 1/S1/S1J/src/s1/factoriaMontana.java
package s1;

import java.util.ArrayList;

public class factoriaMontana implements factoriaCarrerayBicicleta {

	public carreraMontana crearCarrera(int codigo) {
            carreraMontana carrera = new carreraMontana(codigo);
            return carrera;
	}

	public bicicletaMontana crearBicicleta(int id) {
            bicicletaMontana bicicleta = new bicicletaMontana(id);
            return bicicleta;
	}
}